mantis-common/src/main/java/io/mantisrx/common/WorkerConstants.java

```diff
@@ -24,6 +24,6 @@ public class WorkerConstants {
     // TODO(fdichiara): make this configurable.
     public static final String AUTO_SCALE_GROUP_KEY = "NETFLIX_AUTO_SCALE_GROUP";
     public static final String MANTIS_WORKER_CONTAINER_GENERATION = "MANTIS_WORKER_CONTAINER_GENERATION";
-    public static final Pattern MANTIS_SCHEDULING_ATTRIBUTE_PATTERN = Pattern.compile("^MANTIS_SCHEDULING_ATTRIBUTE_(.*)$");
+    public static final Pattern MANTIS_SCHEDULING_ATTRIBUTE_PATTERN = Pattern.compile("^(?i)MANTIS_SCHEDULING_ATTRIBUTE_(.*)$");
```

Comment:
nit: why a case-insensitive match?

Reply:
although the convention seems to be that all task attributes are lower-case, I implemented it this way as a precautionary measure. This is to prevent any possible mismatches, as they could potentially result in a complete absence of task scheduling.